src/content/docs/features/cachyos_settings.mdx

```diff
@@ -0,0 +1,113 @@
+---
+title: CachyOS Settings
+---
+import { Tabs, TabItem, Steps } from '@astrojs/starlight/components';
+
+
+Alongside our optimized kernels and repositories, we also provide settings that further improve the desktop experience.
+We also bundle in some helper scripts to for QoL improvements. All these configurations and scripts are under the `cachyos-settings` package.
+
+## sysctl Tweaks
+
+We provide a lot of sysctl tweaks that aims to improve overall desktop performance. Each sysctl entry is documented
+well in the file [`99-cachyos-settings.conf`](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/sysctl.d/99-cachyos-settings.conf)
+
+To make changes to any of these values, copy the original entry and make a new file under `/etc/sysctl.d/` for the modified value.
+
+### Modifying sysctl values
+
+<Steps>
+
+1. Take a look at the original value from `cachyos-settings`
+
+    ```sh
+    ❯ cat /usr/lib/sysctl.d/99-cachyos-settings.conf
+    # Only experimental!
+    # Let Realtime tasks run as long they need
+    # sched: RT throttling activated
+    kernel.sched_rt_runtime_us=-1
+    ```
+
+2. Make a new file in `/etc/sysctl.d` to make changes to the sysctl settings
+
+    ```sh title="Reverting kernel.sched_rt_runtime_us= to its default value"
+    ❯ sudo micro /etc/sysctl.d/99-kerel-sched-rt.conf # If the file doesn't exist, this command creates and lets you edit the file
+    kernel.sched_rt_runtime_us=950000
+    ```
+
+</Steps>
+
+## udev Rules
+
+- [ZRAM Rules](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/30-zram.rules) - Sets ZRAM swappiness to a more aggressive
+value so cache is more likely to swap to ZRAM
+- [HPET Permissions](https://github.com/CachyOS/CachyOS-Settings/blob/master/usr/lib/udev/rules.d/40-hpet-permissions.rules) - Lower latency audio
```

Comment:
I underlooked this in the last review, but similar to `cpu_dma_latency`, these rules don't do anything other than change permissions.

Reply:
Fixed